staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2758,6 +2761,19 @@ const (
 	Tap DomainAttachmentType = "tap"
 )
 
+type InterfaceBindingMigration struct {
```

Comment:
IMO it is wrong to use the existence of the `migration` vale as a non-nil to represent it as "disabled". The usage of this pattern is not elegant and has several limitation which have been discussed previously (in other threads).

For now, I feel uncomfortable with this, but I will not block on it. At the same time I do not feel I can ack it explicitly.